tests/svntests.py

```diff
@@ -36,3 +36,212 @@ def test_versionformat_revision(self):
         basename = self.basename(version='foo2')
         th = self.assertTarOnly(basename)
         self.assertTarMemberContains(th, basename + '/a', '2')
+
+    def _check_servicedata(self, expected_dirents=2, revision=2):
```

Comment:
Moving shared tests into a new `gitsvntests.py` file was a huge improvement, so thanks a lot for that.  But there is still quite a bit of code duplication being introduced in these helper functions.  For example this `_check_servicedata()` function is almost identical to the one in `gittests.py`.  It should be easy to move the common duplicated parts into `gitsvntests.py` and only keep the differing bits in `gittests.py` and `svntests.py`.